test/joiner_join_project_data_test.rb

```diff
@@ -25,7 +25,8 @@ def test_join_project
         { 'msb-usa' =>
           {
             'name' => 'MSB-USA', 'status' => 'Hold',
-            'self' => project_self_link('msb-usa')
+            'self' => project_self_link('msb-usa'),
+            'categories' => []
```

Comment:
Nah, just leave it be. We're not testing categories behavior; if we need it to build, we need it to build.